website/docs/r/vm_internal_disk.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vm_internal_disk"
+sidebar_current: "docs-vcd-vm-internal-disk"
+description: |-
+  Provides a vCloud Director VM internal disk resource. This can be used to create and delete VM internal disks.
+---
+
+# vcd\_vm\_internal\_disk
+
+Provides a vCloud Director VM internal disk resource. This can be used to create and delete VM internal disks.
```

Comment:
I'd like to make this more specific and link related VM resource. What about this (highlighting changes):

> This can be used to create, **update** and delete VM internal disks **on already created VMs**.
> 
> **Note (yellow block): To adjust disk parameters when creating a new VM, please use override_template_disk[link-to-the-block-in-VM-doc].**
> 
> **To manage disks which already exist inside a VM, please import[link-to-section-below] them first.**

Reply:
added